.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,49 @@
+---
+description: Report a bug.
+labels: [bug]
+name: Bug Report
+title: "<SCOPE>: <SUMMARY>"
+
+body:
+  - type: textarea
+
+    attributes:
+      description: Concise, complete, and comprehesnsive description.
+      label: Description
+
+      placeholder: |-
+        #### Summary
+        #### Minimal Working Example
+        #### Expected vs Actual Behavior
+        #### Logs or Screenshots
+
+  - type: textarea
+
+    attributes:
+      description: >
+        Provide exact input revisions in the following pre-populated table, or
+        replace the entire content with a locked `flake.lock` URL.
+
+      label: Input Versions
+      placeholder: https://github.com/<USER>/<REPOSITORY>/blob/<HASH>/flake.lock
+
+      value: |-
+        |    Input     |                  Version                   |
+        |--------------|--------------------------------------------|
+        | Stylix       | `0000000000000000000000000000000000000000` |
+        | Home Manager | `0000000000000000000000000000000000000000` |
+        | Nixpkgs      | `0000000000000000000000000000000000000000` |
```

Comment:
Formatting the versions as a table is inconvenient and might discourage people from opening issues if they aren't familiar with `flake.lock`.

For simplicity we should just ask users to paste their `flake.lock`. Pasting is better than asking for a link since links can change or break over time, and we should try to preserve the history of issues as much as possible for future reference. Furthermore, we can set `render: json` on the field to automatically create a Markdown code block from the pasted content.